netstandard/ref/System.Diagnostics.cs

```diff
@@ -38,8 +38,6 @@ public static partial class Debug
         public static bool AutoFlush { get { throw null; } set { } }
         public static int IndentLevel { get { throw null; } set { } }
         public static int IndentSize { get { throw null; } set { } }
-// Removed to break dependency on the larger TraceListener set of APIs
```

Comment:
Please keep this commented out API with the comment.

Reply:
Good catch. Fixed.